### PR TITLE
Fixed issue with tags not being parsed, added potential tags to output

### LINKED ISSUE #149: Output git tag in `mach update` output
If a certain commit is tagged, show the git tag next to the commit hash.
Ideally you'd want to work with tags in your component versioning.

Output could be something like;

```
Updates for my-component...
  1.1.0 (765b554): feat: update commercetools SDK
  6952faf: chore: formatting
```

### LINKED ISSUE #279: Process component version when a tag is provided

### PR DESCRIPTION
**Pull request by demeyerthom**
Fixes #149 
Fixes #279 

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

-  Fixed issue with tag not being parsed correctly
- Added linked tags to update output

### CODE REVIEW

File: internal/updater/changes.go [hunk 1]
```diff
@@ -38,6 +39,19 @@ func (cs *ChangeSet) HasChanges() bool {
 	return cs.Component.Version != cs.LastVersion
 }
 
+func generateTagMessage(tags []string) string {
+	if len(tags) == 0 {
+		return ""
+	}
+
+	tagList := strings.Join(tags, ", ")
+	if len(tagList) > 50 {
```
Comment (MichielBijland) [inline internal/updater/changes.go:48]: why 50? Think this could be less like 10 or even 5

File: internal/updater/updates.go [hunk 2]
```diff
@@ -47,54 +49,45 @@ func findUpdatesSerial(ctx context.Context, cfg *PartialConfig, filename string)
...
 
-	var wg sync.WaitGroup
-	var numWorkers = 4
-
-	// Start 4 workers
-	for i := 0; i < numWorkers; i++ {
-		wg.Add(1)
-		go func() {
-			defer wg.Done()
+	var numWorkers = int(math.Ceil(float64(runtime.NumCPU() / 2)))
```
Comment (MichielBijland) [inline internal/updater/updates.go:55]: might be a good idea to have a minimum of 2 workers?

File: internal/updater/changes.go (hunk 1 above)
Comment (demeyerthom) [reply internal/updater/changes.go:48]: its the number of chars. Do you think less than 50 makes sense in that context? For example frasers often has longer tag names as well as multiple tags

File: internal/updater/updates.go (hunk 2 above)
Comment (demeyerthom) [reply internal/updater/updates.go:55]: Yeah, not opposed to that. Will update the worker calc to have 2 as minimum

File: internal/updater/changes.go (hunk 1 above)
Comment (MichielBijland) [reply internal/updater/changes.go:48]: sorry misread that, then it's fine.

**Review by MichielBijland — APPROVED** [review]

**Pull request merged**